src/state/opamFormatUpgrade.ml

```diff
@@ -1074,7 +1074,15 @@ let from_2_1_alpha2_to_v2_1_rc root conf =
 
 let from_2_1_rc_to_v2_1 _ conf = conf
 
-let from_2_0_to_v2_1 _ conf = conf
+let from_2_0_to_v2_1 _ conf =
+  match OpamFile.Config.jobs conf with
+  | Some n when n = OpamSystem.cpu_count () - 1 ->
```

Comment:
Why this condition? If cpu changed meanwhile, it can no more be the good value

Reply:
Ah.. I was tired ^^"